rougelike_game/Game_working.py

```diff
@@ -1,76 +1,157 @@
 
+bot = find_player(level, Player, looking)
 
 
 
+#while True:  # making a loop
+ #   try:  # used try so that if user pressed other than the given key error will not be shown
+  #      if keyboard.is_pressed('q'):  # if key 'q' is pressed
+   #         print('You Pressed A Key!')
+    #        break  # finishing the loop
+    #except:
+     #   break  # if user pressed a key other than the given key the loop will break
 
 while True:
-
-    print(player1.y, player1.x)
     level[player1.y][player1.x] = '.'
-
-    command = input('> Please, write your command: ')
-    sp.call('clear', shell=True)
-    if command == 'exit':
-        print('Goodbye!')
+    level[bot.x][bot.y] = '.'
+    while True:
+        try:
+            if keyboard.is_pressed('e'):
+                print('EXIT y/n')
+                while True:
+                    try:
+                        if keyboard.is_pressed('y'):
+                            print('Goodbye!')
+                            b1 = 2
+                            b2 = 2
+                            sp.call('clear', shell=True)
+                            break
+                        elif keyboard.is_pressed('n'):
+                            b2 = 2
+                            sp.call('clear', shell=True)
+                            break
+                    except:
+                        break
+
+
+
+            elif keyboard.is_pressed('w') and player1.y > 0:
+                if level[player1.y - 1][player1.x] != '#':
+                    player1.move_up()
+                    sp.call('clear', shell=True)
+                    break
+
+            elif keyboard.is_pressed('s') and player1.y < 21:
+                if level[player1.y + 1][player1.x] != '#':
+                    player1.move_down()
+                    sp.call('clear', shell=True)
+                    break
+
+
+            elif keyboard.is_pressed('d') and player1.x < 21:
+                if level[player1.y][player1.x + 1] != '#':
+                    player1.move_right()
+                    sp.call('clear', shell=True)
+                    break
+
+
+            elif keyboard.is_pressed('a') and player1.x > 0:
+                if level[player1.y][player1.x - 1] != '#':
+                    player1.move_left()
+                    sp.call('clear', shell=True)
+                    break
+
+        except:
+            e = 1
+            level[player1.y][player1.x] = '@'
+        if b2 == 2:
+            break
+    level[player1.y][player1.x] = '@'
+    time.sleep(0.1)
+    b2 = 1
+    if b1 == 2:
         break
+    level[player1.y][player1.x] = '@'
 
-    print("You've written " + command)
 
-    if command == 'up' and player1.x > 0:
-        if level[player1.y - 1][player1.x] != '#':
-            player1.move_up()
 
-    elif command == 'down' and player1.x < 20:
-        if level[player1.y + 1][player1.x] != '#':
-            player1.move_down()
+    while True:
+        random_move_generator = random.randint(0, 1000)
```

Comment:
why don't you just generate a random number from 0 to 3? you can even map it to some list of commands, for example:
```
bot_commands = ['up', 'down', 'left' 'right']
choice = random.randint(0, 3)
chosen_command = bot_commands[choice]
```

or even better, use a `random.choice` function
```
bot_commands = ['up', 'down', 'left' 'right']
chosen_command = random.choice(bot_commands)
```